wpeverest/user-registration
# Enhancement - Form field shows upgrade to premium notice

### All Submissions:


### Changes proposed in this Pull Request:

Previously, If users have a user registration pro version and he/she activated ur pro by any license plan. Then in the form builder, on the click of the locked field, it showed Upgrade your plan notice which was not correct according to activated license plan. This PR is changing the notice according to the user's license plan. If the locked field is available in the user's license plan then it notifies as install or activate addon. If not, then upgrade your plan?

### How to test the changes in this Pull Request:

1. First Activate User Registration Pro -> Activate with any license plan key.
2. Then Go to form builder -> click on the locked field -> see the notice message
3. And verify with your plan key, if this field lies under your plan key then it will show install or activate(if the addon already exists) message. If not, then the upgrade plan notice will come.

### Types of changes:


### Other information:


### Changelog entry

>Enhancement - Form field shows upgrade to premium notice
